applications/nftaa.md

```diff
@@ -0,0 +1,411 @@
+the limitations and potential improvements in current systems.
+Case studies examining current issues faced by stakers and asset managers in blockchain networks like Ethereum and
+Polkadot provided practical insights into the challenges and opportunities for innovation.
+Additionally, research on the drawbacks of existing staking mechanisms and the potential benefits of using NFTs for
+account management provided a solid foundation for the project's conceptual framework.
+
+There are similar projects in the Polkadot universe:
+
+- Lido: A liquid staking protocol that allows users to stake assets and receive derivatives representing the staked
+  amount.
+- Bifrost: Provides liquidity for staked assets through a voucher system and supports cross-chain staking.
+- Acala: Offers a similar staking mechanism with its LDOT token, providing immediate liquidity and flexibility in
+  staking.
+
+Differences and Distinctions:
+
+- NFTAA vs. Lido/Bifrost/Acala: Unlike these projects, NFTAA focuses on using NFTs as proxy accounts for staking, which
+  allows for a more flexible and secure delegation of staking rights and ownership. NFTAA also leverages the composability
+  of NFTs, enabling them to hold other assets and act as accounts within the blockchain ecosystem.
+
+Similar concepts exist in other ecosystems:
+
+- Rocket Pool: An Ethereum-based protocol for decentralized staking that allows users to earn rewards without running a
+  full node.
+- EIP-6551 (TBA - token bound accounts): A proposed Ethereum standard to enhance NFTs by equipping them with smart contract capabilities, allowing them to own assets and interact with other contracts.
+
+Differences and Distinctions (NFTAA vs. EIP-6551):
+
+NFTAA is designed specifically for the Polkadot and Substrate ecosystems, utilizing their unique features, such as interoperability and cross-chain communication.
+Additionally, NFTAA’s focus on using NFTs as proxy accounts for staking and asset management offers a novel approach compared to the more traditional liquid staking and smart contract enhancements in the Ethereum ecosystem.
+
+EIP 6551 presents certain limitations. Our solution, e.g., does not prioritize connecting existing NFTs but enables bidirectional binding of the NFT and proxy.
+Consequently, upon ownership of an NFT, we can discern from the NFT metadata indicating its status as an NFTAA.
+Furthermore, we enhance transparency by ensuring the atomic creation of the NFT and proxy within a single transaction, thereby promoting seamless integration.
+Likewise, our design does not need the registrar service to create the TBA address as opposed to TBA.
+In our case, the pallet creates the NFTAA with functionality that also works as the NFTAA address (TBA uses one more address than NFTAA).
+Another difference is that in TBA, you could have more accounts on one NFT, which, in our case, is not possible due to the atomic creation of exactly one NFT at the time of creating the NFTAA.
```

Comment:
The whole substrate runtime is synchronous so I don't see why this "atomic creation" would be unique to your solution.

Reply:
We didn't realize that, to be honest, and so you're right. But atomic creation guarantees same conditions across all parachains and future usages of pallet and moreover it is a guarantee to us that process works exactly as it was designed, so we take it as another safety catch.